skylog/storage/gcp/cloudfunc/go.mod

```diff
@@ -11,6 +11,6 @@ require (
 	github.com/go-toolsmith/pkgload v1.0.0 // indirect
 	github.com/go-toolsmith/typep v1.0.0 // indirect
 	github.com/golang/protobuf v1.3.2-0.20190517061210-b285ee9cfc6c
-	github.com/google/trillian v1.2.2-0.20190619133759-4511faac1a5c
```

Comment:
Are these module changes expected too?

Reply:
Yes, it picks up new additions to the `BuildJob` proto.